### ISSUE TITLE
Registration form is broken

### ISSUE THREAD

**Issue by websocketdev** [bn3t/generator-ngx-firebase-bootstrap#9]
When registering with a new user, get this error in Chrome (Version 68.0.3440.106 (Official Build) (64-bit)):
`login:1 [DOM] Found 2 elements with non-unique id #email: (More info: https://goo.gl/9p2vKq) <input _ngcontent-c4 class=​"form-control ng-untouched ng-pristine ng-invalid" id=​"email" name=​"email" required type=​"text" ng-reflect-required ng-reflect-form=​"[object Object]​">​ <input _ngcontent-c5 class=​"form-control ng-untouched ng-pristine ng-invalid" id=​"email" name=​"email" required type=​"text" ng-reflect-required ng-reflect-form=​"[object Object]​">​
`
And user is automatically logged in, so if navigating back it shows logged in user info with assigned user ID. 

On Firefox (61.0.2 (64-bit), nothing happens when hitting registration button, no registration confirmation, and when navigating back, new user is logged in. There's a warning in the console: 
`Use of Mutation Events is deprecated. Use MutationObserver instead.`

As a whole, looks like the registration form is broken.

When registering an already existing user, form doesn't react either, but navigating back has no one logged in.

**Comment by bn3t** [comment]
Indeed, when registering via the register page, it lacks a redirect to the info page showing the user information. This is done correctly when using registration via the all-in-one page.

Also, indeed the ids used should be cleanup to avoid duplicates.

Globally, the registration form works although not fully user friendly.

I'll fix these issues when I have some time to work on the project again.
